src/Service/EventBridge/EventBridgeTransportFactory.php

```diff
@@ -2,26 +2,28 @@
 
 namespace Bref\Symfony\Messenger\Service\EventBridge;
 
-use Aws\EventBridge\EventBridgeClient;
+use Psr\Container\ContainerInterface;
 use Symfony\Component\Messenger\Transport\Serialization\SerializerInterface;
 use Symfony\Component\Messenger\Transport\TransportFactoryInterface;
 use Symfony\Component\Messenger\Transport\TransportInterface;
 
 final class EventBridgeTransportFactory implements TransportFactoryInterface
 {
-    /** @var EventBridgeClient */
-    private $sns;
+    /** @var ContainerInterface */
+    private $container;
 
-    public function __construct(EventBridgeClient $sns)
+    public function __construct(ContainerInterface $container)
```

Comment:
I guess here we could probably replace that with a "service locator" or "service subscriber" (https://symfony.com/doc/current/service_container/service_subscribers_locators.html#defining-a-service-locator). I don't care too much about that, except that it could remove the need for `public` so why not.

WDYT?